tests/searchcommands/test_searchcommands_app.py

```diff
@@ -227,7 +227,7 @@ def test_sum_as_unit(self):
 
         expected, output, errors, exit_status = self._run_command('sum', action='execute', phase='map', protocol=1)
         self.assertEqual(0, exit_status, msg=six.text_type(errors))
-        self.assertEqual('2019-03-27 20:45:12,006, Level=ERROR, Pi[203 chars]t.', errors)
+        self.assertEqual('2019-03-27 20:45:12,006, Level=ERROR, Pi[203 chars]t.\n', errors)
```

Comment:
this shouldn't be a hardcoded timestamp, use a regex or substring match

Reply:
i reverted back to the previous one and use splunklib handler instead of stderr. I can't reproduce this error locally tho.